test/quantization/core/experimental/apot_fx_graph_mode_ptq.py

```diff
@@ -54,33 +57,46 @@ def calibrate(model, data_loader):
 """
 Prepare model PTQ for specified qconfig for torch.nn.Linear
 """
+
+
 def prepare_ptq_linear(qconfig):
     qconfig_dict = {"object_type": [(torch.nn.Linear, qconfig)]}
-    prepared_model = prepare_qat_fx(copy.deepcopy(float_model), qconfig_dict)  # fuse modules and insert observers
+    prepared_model = prepare_qat_fx(
+        copy.deepcopy(float_model), qconfig_dict
+    )  # fuse modules and insert observers
     calibrate(prepared_model, data_loader_test)  # run calibration on sample data
     return prepared_model
 
+
 """
 Prepare model with uniform activation, uniform weight
 b=8, k=2
 """
 
 prepared_model = prepare_ptq_linear(uniform_qconfig_8bit)
-quantized_model = convert_fx(prepared_model)  # convert the calibrated model to a quantized model  # noqa: F821
+quantized_model = convert_fx(
+    prepared_model
+)  # convert the calibrated model to a quantized model  # noqa: F821
```

Comment:
wrong location of `noqa`, you can run `make seup_lint && make lint` locally, see https://github.com/pytorch/pytorch/blob/main/CONTRIBUTING.md#local-linting for more details

for example:

```suggestion
quantized_model = convert_fx(  # noqa: F821
    prepared_model
)  # convert the calibrated model to a quantized model
```

Reply:
or you can cherry-pick this commit: https://github.com/pytorch/pytorch/pull/126522/commits/96eac508c99738977b8407a3082a7bdc3b89446f